CONTRIBUTING.md

```diff
@@ -255,7 +255,8 @@ For instance:
 ## PRs & releases
 
 The default branch of the repository is `master`, but each regular (non-hotfix, non-release, non-backport) PR must be merged to `develop`. <br/>
-Because of that all regular PR branches should be derived from `develop` and not `master`. <br/>
+Because of that all regular PR branches should start from `develop` and not `master`.  <br/>
+They can also be based on other PR branches as GitHub automatically handles pull request retargeting ([GitHub blogpost](https://github.blog/changelog/2020-05-19-pull-request-retargeting/)).<br/>
```

Comment:
> as

The two facts aren't related.

Even when GitHub did NOT auto-retarget PRs, creating stacked PRs was perfectly possible... you just needed to retarget them manually or via an API call. I'd remove this sentence... see below